### PR TITLE
Intergated Loading Spinner

### PR DESCRIPTION
**Pull request by Nitish814112**
integrated!!

https://github.com/user-attachments/assets/e782a445-0234-4147-80ff-4e80007c2992

### CODE REVIEW

File: src/pages/HomePage.jsx [hunk 1]
```diff
@@ -157,7 +158,7 @@ export default function HomePage() {
   const processedSearchedMovies = movies.map((movie) => setMediaType(movie));
   const processedTvSeries = tvSeries.map((movie) => setMediaType(movie));
 
-  if (moviesLoading) return <p>Loading movies...</p>;
+  if (moviesLoading && tvSeriesLoading) return <LoaderSpinner/>;
   if (moviesError) return <p>Error: {moviesError}</p>;
```
Comment (yashtibrewal) [inline src/pages/HomePage.jsx:163]: handle tv series error as well

File: src/pages/TvSeries/index.jsx [hunk 2]
```diff
@@ -133,15 +153,6 @@ export default function TVSeriesPage() {
     );
   }
 
-  if (searchQuery && searchedTVSeries.length === 0) {
```
Comment (yashtibrewal) [inline src/pages/TvSeries/index.jsx:136]: I see you have removed it, what is the fallback if the `searchQuery` has content, and `searchedTVSeries` has content ?

**Review by yashtibrewal — CHANGES_REQUESTED** [review]

File: src/pages/TvSeries/index.jsx (hunk 2 above)
Comment (Nitish814112) [reply src/pages/TvSeries/index.jsx:136]: i tried to fix not result found issue

File: src/components/tv/index.jsx [hunk 3]
```diff
@@ -38,7 +39,11 @@ function TV() {
   }, [id, state.tmdbToken]);
 
   return loading ? (
-    <div>Loading</div>
+<<<<<<< Updated upstream
+    <LoaderSpinner />
+=======
+   <LoaderSpinner/>
+>>>>>>> Stashed changes
```
Comment (yashtibrewal) [inline src/components/tv/index.jsx:46]: ?

**Review by yashtibrewal — APPROVED** [review]

**Pull request merged**